android/library/core/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/auth/signup/SignUpView.java

```diff
@@ -39,20 +38,14 @@
 
 	public SignUpView(Context context) {
 		super(context);
-
-		DefaultTheme.initIfThemeNotPresent(context);
```

Comment:
I'm afraid that yes, that's Java helping. Constructors can't be inherited and because the base class defines several constructors the empty one is not created by default.

Theoretically we only need one but each SDK version calls a different one :(